### PR TITLE
Add Paulmann 501.34 4-button remote support

### PR DESCRIPTION
**Pull request by MattFromGer**

### CODE REVIEW

File: zhaquirks/paulmann/fourbtnremote.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+                ],
+            },
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.LEVEL_CONTROL_SWITCH,
```
Comment (javicalle) [inline zhaquirks/paulmann/fourbtnremote.py:103]: I believe that there is no need to include `PROFILE_ID` and `DEVICE_TYPE` in the replacement part if no change.

File: zhaquirks/paulmann/fourbtnremote.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                    LevelControl.cluster_id,
+                    LightLink.cluster_id,
+                    OnOff.cluster_id,
+                    Ota.cluster_id,
+                    Scenes.cluster_id,
+                ],
+            },
+            2: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.LEVEL_CONTROL_SWITCH,
```
Comment (javicalle) [inline zhaquirks/paulmann/fourbtnremote.py:124]: Same as above.

Comment (MattFromGer) [reply zhaquirks/paulmann/fourbtnremote.py]: Updated

**Review by javicalle — COMMENTED** [review]
LGTM

**Comment by drueppela** [comment]
From: https://community.home-assistant.io/t/paulmann-501-34-wall-switch/418414/7?u=ajd_ht

---

I paired

Paulmann 501.34 Wall Switch ([Wandschalter Smart Home Zigbee 3.0 On/Off/Dimm Weiß](https://de.paulmann.com/p/wandschalter-smart-home-zigbee-3.0-on-off-dimm-weiss/50134))

with

Home Assistant 2023.10.3
Supervisor 2023.10.0
Operating System 11.0
Frontend 20231005.0 - latest

and

ZHA

I cannot find any possibility to integrate the Paulmann 501.34 Wall Switch in any automation.

I only find 1 device

image


with 4 entities

image

**Comment by TheJulianJES** [comment]
@drueppela Create a new issue.

**Comment by drueppela** [comment]
> @drueppela Create a new issue.

@TheJulianJES & @dmulcahey : I did. Can I do something else? I bought a bunch of Paulmann 501.34 Wall Switch (sw_build_id (id: 0x4000) 2.7.6_r25) and need to decide if I keep them.

**Pull request merged**